extension_templates/classification.py

```diff
@@ -273,6 +274,14 @@ def get_test_params(cls, parameter_set="default"):
         # A parameter dictionary must be returned *for all values* of parameter_set,
         #   i.e., "parameter_set not available" errors should never be raised.
         #
+        # A good parameter set should primarily satisfy two criteria,
+        #   1. Chosen set of parameters should reduce the testing time.
+        #       This is vital for the case of Deep Classifiers where default values
+        #       lead of "big" models which not only increases test time but also
+        #       run into the risk of test workers crashing.
+        #   2. There should be minimum two such parameter set with different values
```

Comment:
bit odd english, perhaps reformulate?

Reply:
I'm confused about whether you're referring to the first point or the second point (or maybe both?)

Reply:
-> there should be *a* minimum
-> parameter *sets*
-> "maximum code coverage" is vague, unspecific